<launch>
	<node pkg="ira_laser_tools" name="laserscan_multi_merger" type="laserscan_multi_merger" output="screen">
	     <param name="destination_frame" value="pal"/>
	     <param name="cloud_destination_topic" value="/dreamvu/pal/odoa/get/scan_cloud"/>
	     <param name="scan_destination_topic" value="/dreamvu/pal/odoa/get/scan"/>
	     <param name="laserscan_topics" value ="/dreamvu/pal/odoa/get/scan1 /dreamvu/pal/odoa/get/scan2" /> 
	     <param name="angle_min" value="-3.14"/>
	     <param name="angle_max" value="3.14"/>
	     <param name="angle_increment" value="0.00478658545762"/>
	     <param name="range_min" value="0.00"/>
	     <param name="range_max" value="50.0"/>
	</node>
</launch>
